### ISSUE TITLE
List_* operations should return actual objects instead of items

### ISSUE THREAD

**Issue by jugi92** [DaSenf1860/ms-fabric-sdk-core#8]
For example list_lakehouses returns a List of Items. It would be easier to work with this list if the list would contain actual lakehouse objects, so that you can directly call lakehouse specfic operations on those.
Currently working with the list would require the following code, which the SDK could abstract:
```
lh_items = ws.list_lakehouses()
lhs = []
for lakehouse_item in lh_items:
    target_lhs.append(ws.get_lakehouse(lakehouse_item.id)

# continue to work with the lakehouse objects.
```

**Comment by DaSenf1860** [comment]
Hi @jugi92 , this is actual already implemented.

You can call list_lakehouses(with_properties = True). Then it return lakehouse objects.

However there is a bug which slipped to my tests. I´ll fix it today

**Comment by DaSenf1860** [comment]
the bug is fixed in version 0.1.6

you can use list_lakehouses(with_properties = True) to return lakehouse objects
